.meteor/versions

```diff
@@ -106,6 +109,8 @@ msavin:mongol@1.6.2
 nooitaf:colors@1.1.2
 npm-bcrypt@0.7.8_2
 npm-mongo@1.4.39_1
+oauth@1.1.6
+oauth2@1.1.5
```

Comment:
I didn't add both. It's probably a dependency or something. Wala if it's in the versions it means that I downloaded it? because I don't remember doing that